docs/ja/user-guide/configuration/build-cache.md

```diff
@@ -16,9 +16,9 @@ toc:
 
 | スコープ  | アクセス範囲 |
 |---|---|
-| pipeline  | 同じパイプラインの全てのビルド  |
-| event  | 同じイベントの全てのビルド  |
-| job  | 同じジョブについての全てのビルド  |
+| pipeline  | 同じパイプラインの全てのビルド ![pipeline-scope](../../ja/user-guide/assets/pipeline-scope.png) |
```

Comment:
Isn't `ipeline-scope.png` in the `../../../user-guide/assets/` folder?

Reply:
Fix path.